semperfiwebdesign/all-in-one-seo-pack
# Fix undefined $tag in aioseop_opengraph.php

Issue #1688

## Proposed changes

Fixes a user-reported bug where the 'tag' variable isn't defined in aioseop_opengraph.php. 

This happens when "Automatically Generate Article Tags" in the Social Meta menu is enabled but "Use Keywords In Article Tags" isn't.

## Types of changes

- Bugfix (non-breaking change which fixes an issue)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._


## Testing instructions

1. Go to the General Settings menu and enable Meta Keywords.
2. Create a new Post, assign it a Category, Tag and give it one or more meta keywords.
3. Go to the Social Meta menu and enable  "Automatically Generate Article Tags" but disable "Use Keywords In Article Tags".
4. Enable WP_DEBUG in the wp-config file.
5. View the Post and make sure that the "undefined tag variable in aioseop_opengraph.php" notice is no longer present.

## Further comments

You can reproduce the notice using the same steps and the current release of the free version.
